larsim/EventGenerator/SingleGen_module.cc

```diff
@@ -447,11 +448,11 @@ namespace evgen {
     , fHistFileName(config().HistogramFile())
     , fPHist(config().PHist())
     , fThetaXzYzHist(config().ThetaXzYzHist())
-    , fEngine(createEngine(0))
+    , fEngine(art::ServiceHandle<rndm::NuRandomService>()
+                ->registerAndSeedEngine(createEngine(config().Seed()), "HepJamesRandom", "Seed"))
```

Comment:
@mantheys, let's try @PetrilloAtWork's suggestion:

```suggestion
                ->registerAndSeedEngine(createEngine(config().Seed()), "", "", config().Seed()))
```